Example run

deanlorenz opened the pull request:
README for using `run.sh` with an existing stack.

maugustosilva commented on quickstart-existing-stack-benchmark/run_example.md:
```diff
@@ -0,0 +1,237 @@
+
+### 3. Gather required parameters (mostly information about your `llm-d` stack)
+
+* **Namespace**:
```
By the default, the **current** `namespace` will be used. Couldn't we make it optional?

maugustosilva replied:
#288 got merged. Can you give it a try and let me know? Thanks!

deanlorenz replied:
> #288 got merged. Can you give it a try and let me know? Thanks!

Still did not work for me. Uses default namespace instead of current.

maugustosilva commented on quickstart-existing-stack-benchmark/run_example.md:
````diff
@@ -0,0 +1,237 @@
+  ```
+  </details>
+
+* **Model**: 
````
If `-t` is **not** `modelservice` and **not** `standalone` , then the model is set to `auto` and automatically discovered.

deanlorenz replied:
Great. I will remove it.

maugustosilva commented:
Many thanks for this. Trying to make even simpler for a "first time user" by omitting some of the parameters.

deanlorenz commented on quickstart-existing-stack-benchmark/run_example.md:
````diff
@@ -0,0 +1,237 @@
+  ```
+  </details>
+
+* **Hugging-Face Token**: 
````
I will change the wording to make it a recommendation. When are we actually using it in `run.sh`?

deanlorenz commented on quickstart-existing-stack-benchmark/run_example.md:
```diff
@@ -0,0 +1,237 @@
+  </details>
+
+* **PVC X2**: 
+  2 Persistent Volume Claims; one _Common PVC_ for storing the model and cache and one _Harness PVC_ for storing benchmarking results. Must be one of the available PVCs in the cluster.
```
We have `model-pvc` and `workload-pvc`. I think they cannot share the same PVC. We tried `run.sh` before without `model-pvc` and it did not work. I will test again. For now, I removed the requirement from the readme.

deanlorenz commented:
- Updated the README and moved to docs
- Added wizard script to help create the environment variables
  - More explicit defaults that the user can change
- Tested with `infra` and on both mac and linux
  - Optional parameters seem to work
  - Namespace must still be set (discovery does not work with current context).
